src/main/webapp/app/lecture/pdf-preview/pdf-preview.component.ts

```
@@ -334,8 +450,169 @@
+    async deleteAttachmentFile() {
+        if (this.attachment) {
+            this.attachmentService.delete(this.attachment.id!).subscribe({
+                next: () => {
+                    this.router.navigate(['course-management', this.course?.id, 'lectures', this.attachment!.lecture!.id, 'attachments']);
+                    this.dialogErrorSource.next('');
+                },
+                error: (error) => {
+                    this.alertService.error('artemisApp.attachment.pdfPreview.attachmentUpdateError', { error: error.message });
+                },
+            });
+        } else if (this.attachmentUnit && this.attachmentUnit.id && this.attachmentUnit.lecture?.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)